Formula/nmrpflash.rb

```diff
@@ -0,0 +1,22 @@
+class Nmrpflash < Formula
+  desc "Netgear Unbrick Utility"
+  homepage "https://github.com/jclehner/nmrpflash"
+  url "https://github.com/jclehner/nmrpflash/archive/refs/tags/v0.9.18.tar.gz"
+  sha256 "e1bc46445ab302d76fc4d9a99885f5df404b77a7c836bf2ef9f08fd73390db38"
+  license "GPL-3.0-or-later"
+
+  uses_from_macos "libpcap"
+
+  # for now
+  depends_on :macos
```

Comment:
Why?

Reply:
On Linux, it requires `libnl-route`, for which there's no Homebrew package, so how would you specify that dependency?

Reply:
You'll need to make that package first.

Reply:
Is this `libnl-route`? https://github.com/thom311/libnl

I'm ok to leave it to interested Linux users to package `libnl-route` if they want this formula on Linux. This should be documented in a comment, at least, though.

Reply:
> Is this `libnl-route`? https://github.com/thom311/libnl

Actually, this is already a formula called `libnl`. It provides a `libnl-route-3.so`. Is that not what you need?

Reply:
You're right, I had missed that, because it didn't show up on https://formulae.brew.sh/. I've fixed it now, and verified that it compiles on my Ubuntu machine.